content/en/docs/ambient/usage/l4-policy/index.md

```diff
@@ -75,6 +75,31 @@ Even though the identity of the pod is otherwise correct, the presence of a L7 p
 command terminated with exit code 56
 {{< /text >}}
 
+### Authorization policy attachment
+
+In the simplest enforcement scenario, you want to enforce policy against TCP attributes and you have no waypoint proxies in your traffic's path. These policies can be enforced by ztunnel proxies.
+
+Once you introduce a waypoint proxy, the ideal place to enforce policy shifts. Traffic arriving at the destination ztunnel will be coming from the waypoint's identity because waypoint proxies do not impersonate `src` identity on behalf of the client. This means that even if you only wish to enforce policy against TCP attributes, you should bind that policy to your waypoint proxy. An additional TCP policy may be applied to your workload to request that ztunnel enforce rules like, "in-mesh traffic needs to come from my waypoint in order to reach my application". This type of policy allows you to choose if "bypassing" the waypoint proxy is permissible in your scenario.
```

Comment:
replace 'be coming' with 'come' or 'appear' or 'is presented as' (to make active voice)

> you should bind that policy to your waypoint proxy

Is there any performance penalty?  I guess the implication is "it will go through the waypoint anyway, if the PEP is the destination ztunnel"

Reply:
The point I was trying to make is that if you want to base any of your policy on client identity waypoint is your only real choice. Otherwise even a client called `evil-do-not-ever-allow` will just look like your waypoint proxy to the dest ztunnel so you can't make a very good decision based on identity anymore.